### ISSUE TITLE
[ACLTUAL BUG REPORT LOL] Missing Conditional Compile Statement in VideoState.hx breaks HTML5 builds

### ISSUE THREAD

**Issue by SuperPositiveP** [GrowtopiaFli/openfl-haxeflixel-video-code#10]
The offending code is at line **105** of **VideoState.hx** and not using
```
				#if !web
				vidSound.time = vidSound.length * soundMultiplier;
				#end
```
or
```
				#if desktop
				vidSound.time = vidSound.length * soundMultiplier;
				#end
```
instead of
```
				vidSound.time = vidSound.length * soundMultiplier;
```
breaks things on HTML5 builds. I'm not sure if this line was intended to be commented out or not, so I just did this (maybe I tried commenting it out once and it broke desktop builds...?). At the very least, this breaks HTML5 builds that load the videostate before the playstate by using `LoadingState.loadAndSwitchState(new VideoState('/path/to/webm'), new PlayState()), true);`

(This is a way of loading the videostate which I might, on a side note, recommend adding to the readme since the only video on this is telling people to use FlxG.switchState which WILL break things based on the same video lol.)

**Comment by SuperPositiveP** [comment]
Too lazy to fork and make a PR for literally two lines of changes, sorry lol.

**Comment by SuperPositiveP** [comment]
Got unlazy and made the PR fix in PR #11.

**Comment by GrowtopiaFli** [comment]
yes my bad
i will fix this in 1 week or something
